### PR TITLE
BZ1771435 gather data for restricted install

### PR DESCRIPTION
**Pull request by lamek**
https://bugzilla.redhat.com/show_bug.cgi?id=1771435

### CODE REVIEW

File: modules/support-gather-data.adoc [hunk 1]
```diff
@@ -27,6 +27,29 @@ $ oc adm must-gather
...
 ====
++
+[NOTE]
+====
+If your cluster is using a restricted network you must import the default must-gather image, or mirror the latest must-gather image, and then run the `oc adm must-gather` command.
+
+Importing the default must-gather image:
+
+----
+oc import-image <image stream> must-gather -n openshift
```
Comment (zhouying7780) [inline modules/support-gather-data.adoc:38]: should be `oc import-image is/must-gather -n openshift`

**Comment by zhouying7780** [comment]
Other lgtm, thanks.

File: modules/support-gather-data.adoc [hunk 2]
```diff
@@ -27,6 +27,29 @@ $ oc adm must-gather
 ====
 If this command fails, for example if you cannot schedule a Pod on your cluster, then use the `oc adm inspect` command to gather information for particular resources. Contact Red Hat Support for the recommended resources to gather.
 ====
++
+[NOTE]
+====
+If your cluster is using a restricted network you must import the default must-gather image, or mirror the latest must-gather image, and then run the `oc adm must-gather` command.
```
Comment (kalexand-rh) [inline modules/support-gather-data.adoc:33]: If your cluster is in a restricted network, you probably don't have internet access to run this command. You probably need to actually mirror this content onto the machine that hosts the registry.

File: modules/support-gather-data.adoc [hunk 3]
```diff
@@ -27,6 +27,29 @@ $ oc adm must-gather
...
+
+Importing the default must-gather image:
+
+----
+oc import-image is/must-gather -n openshift
+
+oc adm must-gather
+----
+
+Mirroring the latest must-gather image:
```
Comment (kalexand-rh) [inline modules/support-gather-data.adoc:43]: Don't you need to do this in your private registry?

File: modules/support-gather-data.adoc [hunk 4]
```diff
@@ -27,6 +27,29 @@ $ oc adm must-gather
...
 ====
++
+[NOTE]
+====
+If your cluster is using a restricted network you must import the default must-gather image, or mirror the latest must-gather image, and then run the `oc adm must-gather` command.
+
+Importing the default must-gather image:
+
+----
+oc import-image is/must-gather -n openshift
```
Comment (kalexand-rh) [inline modules/support-gather-data.adoc:38]: All your commands need to start with "$"

File: modules/support-gather-data.adoc [hunk 5]
```diff
@@ -27,6 +27,29 @@ $ oc adm must-gather
...
+
+oc adm must-gather
+----
+
+Mirroring the latest must-gather image:
+
+----
+oc image mirror quay.io/openshift/origin-must-gather:latest \
```
Comment (kalexand-rh) [inline modules/support-gather-data.adoc:46]: All of these commands shouldn't be in a note. Please separate these into different sections. I think that the instructions to mirror the content should go in https://docs.openshift.com/container-platform/4.2/installing/install_config/installing-restricted-networks-preparations.html and that mirroring the content should be a prereq.

**Review by kalexand-rh — COMMENTED** [review]
Please double-check with QE. I question if these instructions will get the user where they need to be.

**Comment by vikram-redhat** [comment]
@lamek is this ok to merge?

**Comment by lamek** [comment]
CP for 4.6
https://github.com/openshift/openshift-docs/pull/24628

CP for 4.5
https://github.com/openshift/openshift-docs/pull/24629

**Pull request merged**